feature/home/src/main/java/com/mashup/dorabangs/feature/home/HomeViewModel.kt

```diff
@@ -343,7 +420,13 @@ class HomeViewModel @Inject constructor(
         setVisibleDialog(false)
         if (response.isSuccess) {
             intent {
-                postSideEffect(HomeSideEffect.RefreshPostList)
+                postSideEffect(HomeSideEffect.DeletePost(postId))
+                idListCache[getCacheKey(state)] =
+                    idListCache[getCacheKey(state)]?.filterNot { it == postId } ?: emptyList()
+                idListCache["all"] =
+                    idListCache["all"]?.filterNot { it == postId } ?: emptyList()
+                idListCache["favorite"] =
+                    idListCache["favorite"]?.filterNot { it == postId } ?: emptyList()
```

Comment:
매번 3개를 해줘야하는게 쉽지않긴하네,,,,ㅋㅋㅋㅋ

Reply:
그러겡 로직 같으면 그냥 하나로 합쳐버려
```suggestion
                   listOf(getCacheKey(state), idListCache["all"], idListCache["favorite"]).map {
        it?.filterNot { it == postId } ?: emptyList()
    }
```
코드만 대충 친거니까 저런식으로 하나로 합치던,, 함수로 뺴던,,, 더 쉽게 만들 . 순있을듯

Reply:
수정 완 ~